README.md

```diff
@@ -1 +1,30 @@
+<h3 align="center">A passionate javaScript developer from Ukraine</h3>
+
+<p align="left"> <img src="https://komarev.com/ghpvc/?username=andrij-maksymyshyn&label=Profile%20views&color=0e75b6&style=flat" alt="andrij-maksymyshyn" /> </p>
+
+- 🔭 I’m currently working on [real-time-chat](https://github.com/Andrij-Maksymyshyn/real-time-chat)
+
+- 🌱 I’m currently learning **Node.js**
+
+- 🤝 I’m looking for help with [Oleksij Chestnyh (mentor)](https://github.com/chestnyh)
+
+- 👨‍💻 All of my projects are available at [https://github.com/Andrij-Maksymyshyn](https://github.com/Andrij-Maksymyshyn)
+
+- 💬 Ask me about **react.js, Node.js, socket.io**
+
+- 📫 How to reach me **maxandrij@ukr.net**
+
+<h3 align="left">Connect with me:</h3>
+<p align="left">
+<a href="https://linkedin.com/in/https://www.linkedin.com/in/andrewmax777/" target="blank"><img align="center" src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg" alt="https://www.linkedin.com/in/andrewmax777/" height="30" width="40" /></a>
+</p>
+
+<h3 align="left">Languages and Tools:</h3>
+<p align="left"> <a href="https://www.w3schools.com/css/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/css3/css3-original-wordmark.svg" alt="css3" width="40" height="40"/> </a> <a href="https://expressjs.com" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/express/express-original-wordmark.svg" alt="express" width="40" height="40"/> </a> <a href="https://www.w3.org/html/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/html5/html5-original-wordmark.svg" alt="html5" width="40" height="40"/> </a> <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/javascript/javascript-original.svg" alt="javascript" width="40" height="40"/> </a> <a href="https://nodejs.org" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/nodejs/nodejs-original-wordmark.svg" alt="nodejs" width="40" height="40"/> </a> <a href="https://reactjs.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original-wordmark.svg" alt="react" width="40" height="40"/> </a> </p>
+
+<p><img align="left" src="https://github-readme-stats.vercel.app/api/top-langs?username=andrij-maksymyshyn&show_icons=true&locale=en&layout=compact" alt="andrij-maksymyshyn" /></p>
+
+<p>&nbsp;<img align="center" src="https://github-readme-stats.vercel.app/api?username=andrij-maksymyshyn&show_icons=true&locale=en" alt="andrij-maksymyshyn" /></p>
+
+<p><img align="center" src="https://github-readme-streak-stats.herokuapp.com/?user=andrij-maksymyshyn&" alt="andrij-maksymyshyn" /></p>
```

Comment:
You should keep this information in different repository. Create repository and name it Andrij-Maksymyshyn. And then this information will be visible in your github page.

This README.md should contain information about your project - some description, how to configure it how, how to run it locally etc.

Reply:
Hi, Oleksij!
This is a temporary README.md. At the end of the project, I will definitely make a nice README.md file.

Reply:
You should describe how to run your project for other developers. You should fill this information during development process. Not in the end of the project. Because for good project there is no end)).